src/table/builder.rs

```diff
@@ -175,7 +195,7 @@ impl Builder {
         let len = res.len();
         assert!(len < u32::MAX as usize);
         self.buf.put_slice(&res);
-        self.buf.put_u32(len as u32);
+        self.buf.put_u32_le(len as u32);
```

Comment:
I think this uses `BigEndian`.

Reply:
Yes, you're right. I read badger `builder.go` again and find out that all builder encoding / decoding is using big endian. And for those using flatbuffers, they're little endian by default. I'll try making these compatible in future PRs.

Reply:
except that:
* `Header`, in badger it directly copy it to data, so it uses system endian.
* Vector data, for example, `offset`, it also uses system endian.

FYI, https://github.com/dgraph-io/badger/blob/master/y/y.go